dfx.json

```diff
@@ -59,6 +59,10 @@
 		"staging": {
 			"providers": ["https://icp0.io"],
 			"type": "persistent"
+		},
+		"local": {
+			"bind": "127.0.0.1:4943",
+			"type": "ephemeral"
```

Comment:
Interesting.  I didn't know that this opt-out still existed.  Perhaps that is why `dfx stop` doesn't always work as expected.  I'll have to test to see whether this is the cause...

Reply:
According to the SDK team, this should solve my problems. I switch between DFX-based projects for the foundation on a weekly basis, and I keep encountering issues with canisters feeling corrupted or disappearing. It turns out that without opting out, there is a central cache for all canisters. Since I'm using the same canister, like II, in multiple projects, I experience these annoying effects when I happen to redeploy one of the projects.